web_m2x_options/__openerp__.py

```diff
@@ -2,13 +2,12 @@
 
 {
     "name": 'web_m2x_options',
-    "version": "8.0.0.2.0",
+    "version": "9.0.0.1.0",
     "depends": [
         'base',
         'web',
     ],
     'data': ['views/view.xml'],
     "author": "0k.io,Odoo Community Association (OCA)",
-    'installable': False,
-    "active": False,
+    'installable': True,
```

Comment:
Better not to do it, because it's used when migrating branches through versions.